src/cli/mod.rs

```diff
@@ -48,6 +55,10 @@ We would love it if you could star the repository (https://github.com/surrealdb/
 struct Cli {
 	#[command(subcommand)]
 	command: Commands,
+	#[arg(help = "Whether to allow web check for client version upgrades at start")]
+	#[arg(env = "SURREAL_CALL_HOME", long)]
+	#[arg(default_value_t = true)]
+	call_home: bool,
```

Comment:
I think this is too generic and can imply a lot of things we don't do. How about naming this flag `--online-version-check` or something like that? Something which actually gives a user a good idea of what it does without having to read the description first.